File: examples/charts/chart.c
#include <stdio.h>

int main(void)
{
    puts("{\"schema\":\"sum.chart/1\",\"kind\":\"bar\",\"title\":\"Shared chart from C\",\"categories\":[\"A\",\"B\",\"C\"],\"series\":[{\"name\":\"value\",\"values\":[25,50,35],\"x_values\":[]}]}");
    return 0;
}

File: examples/charts/chart.rb
#!/usr/bin/env ruby
require "json"
chart = {
  schema: "sum.chart/1",
  kind: "bar",
  title: "Shared chart from Ruby",
  categories: ["A", "B", "C"],
  series: [{name: "value", values: [25, 50, 35], x_values: []}],
}
puts JSON.generate(chart)
